### DISCUSSION TITLE
Lost Password

### DISCUSSION THREAD

**Discussion by Meupets** [roleoroleo/yi-hack-MStar#296] [Q&A]
Hi,
i have use your hack for my Yi home 1080P BFUS but now i have lost my password for use web interface and RTSP flux. Have you got a solution?
Thx

**Comment by roleoroleo** [comment]
Login with ssh and edit manually /home/yi-hack/etc/system.conf

**Comment by Meupets** [comment]
Putty is a good solution for ssh? 
![image](https://user-images.githubusercontent.com/76450854/102888673-8b9c8180-4459-11eb-9803-4cdc6c5a897e.png)

when i try i have an error
![image](https://user-images.githubusercontent.com/76450854/102888729-a242d880-4459-11eb-8528-6e2940f5f9b5.png)

**Comment by roleoroleo** [comment]
Yes but the port is the ssh standard: 22.

**Comment by Meupets** [comment]
Yes now it's work but i need password too :/
![image](https://user-images.githubusercontent.com/76450854/102890696-31052480-445d-11eb-8361-d87482f5838a.png)

**Comment by roleoroleo** [comment]
Did you change ssh password also?
Does blank password work?

**Comment by roleoroleo** [comment]
If you don't find your password, please downgrade to the previous version of the hack and upgrade again.
This will erase completely your config (and all your passwords).
